### ISSUE TITLE
Remember port when restarting after update

### ISSUE THREAD

**Issue by tacomanator** [jhen0409/react-native-debugger#176]
React Native Debugger app version: 0.7.12
React Native version: 0.49.5
Platform: iOS
Is real device of platform: No
Operating System: iOS 11.1

Minor issue. I open `react-native-debugger` with the debugger port used by `create-react-native-app` (```open 'rndebugger://set-debugger-loc?port=19001'```). After updating `react-native-debugger` it prompts to restart, but does not use the same port.

**Comment by jhen0409** [comment]
Yeah currently it doesn't persist the port when app quit, PR welcome for that. 👍
